<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>725</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>725</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>990</width>
    <height>725</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Janela Principal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #222433;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>185</y>
      <width>811</width>
      <height>336</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>20</number>
     </property>
     <item row="2" column="4">
      <widget class="QPushButton" name="QT_voltar">
       <property name="maximumSize">
        <size>
         <width>172</width>
         <height>34</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Voltar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="QT_FPS">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>FPS</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="QT_jogo">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Modo de Jogo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="QT_btPerspectiva">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Perspectiva</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="QT_btVisualizacao">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Visualização</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="QT_Save">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Salvar Padrão</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="QT_btcontrole">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Config. Controle</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="QT_btCores">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Cores</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="QT_Versao">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Versão do QT</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="QT_btCruzetas">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Cruzetas</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="QT_Centro">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Centros e Ângulos</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="QT_Load">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Carregar Padrão</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="QT_btkmedians">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Kmeans</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="QT_btJuiz">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Juiz</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
